files/ruby_plugin_helper.rb

```diff
@@ -0,0 +1,46 @@
+require 'bolt/util'
```

Comment:
The inventory plugin modules explicitly redefine the util methods (i think to avoid requiring bolt library files). https://github.com/puppetlabs/puppetlabs-aws_inventory/blob/d03eebbbe02b60141ce696a2642d18b39fbf621b/tasks/resolve_reference.rb#L79-L97 Should we do that here?

Reply:
Probably, especially if this is going to be language agnostic.